src/msg/async/rdma/RDMAConnectedSocketImpl.cc

```diff
@@ -257,31 +257,29 @@ void RDMAConnectedSocketImpl::handle_connection() {
 
 ssize_t RDMAConnectedSocketImpl::read(char* buf, size_t len)
 {
+  ssize_t read_size = 0;
   eventfd_t event_val = 0;
   int r = eventfd_read(notify_fd, &event_val);
   ldout(cct, 20) << __func__ << " notify_fd : " << event_val << " in " << my_msg.qpn << " r = " << r << dendl;
   
-  if (!active) {
-    ldout(cct, 1) << __func__ << " when ib not active. len: " << len << dendl;
+  if (!active || 0 == connected) {
+    ldout(cct, 1) << __func__ << " ib not active. len: " << len << dendl;
     return -EAGAIN;
-  }
-  
-  if (0 == connected) {
-    ldout(cct, 1) << __func__ << " when ib not connected. len: " << len <<dendl;
+  } else if (0 == connected) {
+    ldout(cct, 1) << __func__ << " ib not connected. len: " << len << dendl;
     return -EAGAIN;
   }
-  ssize_t read = 0;
   if (!buffers.empty())
-    read = read_buffers(buf,len);
+    read_size = read_buffers(buf, len);
```

Comment:
i'd suggest have a separated commit to do the variable renaming and have smaller commits instead of composing a single fat one with multiple changes.

Reply:
@tchaikov I rename the variable when solving problems in some functions. Is it feasible to submit one patch to only change the variable name?

Reply:
as long as your changes are not dependent on each other, and the name changing one does improve the readability, i think it'd be fine.